libs/newsletter/src/driver/hubspot/hubspot.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpTestingController, HttpClientTestingModule } from '@angular/common/http/testing';
+import { TestBed } from '@angular/core/testing';
+import { DaffHubspotNewsletterService } from './newsletter.service';
+import { DaffNewsletterHubSpotDriverModule } from './hubspot-driver.module';
+import { of } from 'rxjs';
+
+describe('DaffHubspotNewsletterDriver', () => {
```

Comment:
This looks like a great test, but its an integration test, not a unit test. It should be moved to an integration test folder and you should specifically unit test the service in question.

Reply:
78d941c09c0116264a0a369fe8e367926d2f78c8